README.md

```diff
@@ -19,14 +19,39 @@ The main features include:
 
 Read more about it on the [OpenRocket Wiki](http://wiki.openrocket.info).
 
+Running OpenRocket
+------------------
+To start the software from the JAR file run:
+```
+$ java -jar OpenRocket-<VERSION>.jar
+```
+
 License
 -------
 
-OpenRocket is an Open Source project licensed under the [GNU GPL](http://openrocket.sourceforge.net/license.html). This means that the software is free to use for whatever purposes, and the source code is also available for studying and extending.
+Copyright (C) 2007-2020  Sampo Niskanen and others
```

Comment:
Honestly, this just seem redundant with the "Contributors" Section Below.

As the GPL doesn't require attributation, it's not particularly relevant.

Reply:
Removed copyright declaration.